blockmanager.go

```diff
@@ -320,6 +332,13 @@ type blockManager struct {
 	startHeader      *list.Element
 	nextCheckpoint   *chaincfg.Checkpoint
 
+	// These fields are related to handling of orphan blocks.  They are
+	// protected by a combination of the chain lock and the orphan lock.
```

Comment:
Aren't the orphan related fields only protected by the orphan lock at this point?

Reply:
Yes, you're correct.  I copied the comment over and didn't update it.  Updated.